EIPS/eip-1.md

```diff
@@ -1,101 +1,100 @@
+  - **ERC** - [FLAG - WE NEED TO MAKE AN EQUIVALENT NAME FOR ERC] application-level standards and conventions, including contract standards such as token standards, name registries, URI schemes, library/package formats, and wallet formats.
+- An **Informational EEP** describes an EOS design issue, or provides general guidelines or information to the EOS community, but does not propose a new feature. Informational EEPs do not necessarily represent EOS community consensus or a recommendation, so users and implementers are free to ignore Informational EEPs or follow their advice.
+- A **Meta EEP** describes a process surrounding EOS or proposes a change to (or an event in) a process. Process EEPs are like Standard Track EEPs but apply to areas other than the EOS protocol itself. They may propose an implementation, but not to EOS's codebase; they often require community consensus; unlike Informational EEPs, they are more than recommendations, and users are typically not free to ignore them. Examples include procedures, guidelines, changes to the decision-making process, and changes to the tools or environment used in EOS development. Any meta-EEP is also considered a Process EEP.
 
-It is highly recommended that a single EIP contain a single key proposal or new idea. The more focused the EIP, the more successful it tends to be. A change to one client doesn't require an EIP; a change that affects multiple clients, or defines a standard for multiple apps to use, does.
+It is highly recommended that a single EEP contain a single key proposal or new idea. The more focused the EEP, the more successful it tends to be. A change to one client doesn't require an EEP; a change that affects multiple clients, or defines a standard for multiple apps to use, does.
 
-An EIP must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate the protocol unduly.
+An EEP must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate the protocol unduly.
 
-## EIP Work Flow
+## EEP Work Flow
 
-Parties involved in the process are you, the champion or *EIP author*, the [*EIP editors*](#eip-editors), and the [*Ethereum Core Developers*](https://github.com/ethereum/pm).
+Parties involved in the process are you, the champion or *EEP author*, the [*EEP editors*](#eep-editors), and the [*EOS Core Developers*]. [FLAG - WE WILL NEED TO MAKE A NEW HASHTAG TO REPLACE #eep-editors AND WE DON'T HAVE CORE DEVS]
```

Comment:
> [FLAG - WE WILL NEED TO MAKE A NEW HASHTAG TO REPLACE #eep-editors AND WE DON'T HAVE CORE DEVS]

`#eep-editors` refers to a section of this document (line 163 of this document) so in this form, it's good.

For the core developers, agree, we will need to come up with our own wordings of who are the core developers of the EOS platform in general.